NewForce-Cohort-5/tabloidfullstack-actual-spring
# Jam delete category

Trello #1 Delete a category https://trello.com/c/qFSjSjuL/11-11-delete-a-category

This code will allow you to click the category navbar and be directed to the category page which then lists the categories from the database in alphabetic order. You should be able to click the add new category button and be directed to an add page where you can create your own category and click save. You should then be taken back to the category page where you can check for your addition,  then click the delete button where a modal should display and allow you to delete the newly added category.

Delete methods were added to the category repository, interface, and controller on the back end side.
The category card was updated to reflect a modal from bootstrap, also methods for deleting the category by id were placed here. Fetch calls for delete category were added to the category provider. 

Type of change
New feature (non-breaking change which adds functionality).
This change requires a documentation update.

Testing Instructions
Reference How to test a teammate's PR
Login using foo@bar
Go to the category management tab on the navbar
You should be able to view the categories listed in alphabetic order.
Click add new category, then type in test category, hit save, look for new category in the list.
Click the delete button on that category or any other category and check to see if it was deleted.

Make sure to add all seed data before testing.
Run this command below to verify the new category was added to the database.
Select * From Category

Please cross check database to browser to ensure the right information is being displayed properly.
